src/snowflake/snowpark/functions.py

```diff
@@ -3019,28 +3220,112 @@ def array_to_string(array: ColumnOrName, separator: ColumnOrName) -> Column:
 
 def object_agg(key: ColumnOrName, value: ColumnOrName) -> Column:
     """Returns one OBJECT per group. For each key-value input pair, where key must be a VARCHAR
-    and value must be a VARIANT, the resulting OBJECT contains a key-value field."""
+    and value must be a VARIANT, the resulting OBJECT contains a key-value field.
+
+    Example::
+
+        >>> from snowflake.snowpark.types import StructType, StructField, VariantType, StringType
+        >>> df = session.create_dataframe(
+        ...     [["name", "Joe"], ["zip", "98004"]],
+        ...     schema=StructType([StructField("k", StringType()), StructField("v", VariantType())])
+        ... )
+        >>> df.select(object_agg(col("k"), col("v")).alias("result")).show()
+        --------------------
+        |"RESULT"          |
+        --------------------
+        |{                 |
+        |  "name": "Joe",  |
+        |  "zip": "98004"  |
+        |}                 |
+        --------------------
+        <BLANKLINE>
+    """
     k = _to_col_if_str(key, "object_agg")
     v = _to_col_if_str(value, "object_agg")
     return builtin("object_agg")(k, v)
 
 
 def object_construct(*key_values: ColumnOrName) -> Column:
-    """Returns an OBJECT constructed from the arguments."""
+    """Returns an OBJECT constructed from the arguments.
+
+    Example::
+
+        >>> from snowflake.snowpark.types import StructType, StructField, VariantType, StringType
+        >>> df = session.create_dataframe(
+        ...     [["name", "Joe"], ["zip", "98004"]],
```

Comment:
Could we have a row with `NULL` key and another row with `NULL` value? So the difference between `object_construct` and `object_construct_keep_null` is obvious?

Reply:
sure, code updated